src/Microsoft.Framework.CommandLineUtils/CommandLine/AnsiConsole.cs

```diff
@@ -9,22 +9,50 @@ namespace Microsoft.Framework.Runtime.Common.CommandLine
 {
     internal class AnsiConsole
     {
-        private AnsiConsole(TextWriter writer)
+        private AnsiConsole(TextWriter writer, IRuntimeEnvironment runtimeEnv)
         {
             Writer = writer;
-            OriginalForegroundColor = Console.ForegroundColor;
+
+            var useColors = runtimeEnv.OperatingSystem == "Windows" || runtimeEnv.RuntimeType == "Mono";
+
+            if (useColors)
+            {
+                WriteLine = WriteLineParse;
+                OriginalForegroundColor = Console.ForegroundColor;
+            }
+            else
+            {
+                WriteLine = WriteLineNoParse;
+            }
         }
 
         private int _boldRecursion;
```

Comment:
Not enforced but it is pretty common to structure a class like this:

- Fields
- Constructors
- Properties
- Methods